vuepress/docs/next/docs/releases/README.md

```diff
@@ -1,10 +1,197 @@
+* The ECR client now explicitly marks bundles that are installed with an amended installation plan and allows the plan to be recalled.
+* Added support for Oracle 18c.
+* Fixed a bug that caused the ECR to indefinitely stay in busy-state due to particular failure conditions.
+* Fixed a bug that (under certain conditions) prevented the bundle plugin pod from properly restarting when scaled down, then back up again.
+* The ECR now correctly deletes the plugin pod when the bundle is uninstalled, but the deployment and volumes are retained.
+
+### ENT (entando-cli)
+
+* Reworked the `ent diag` command to now also export CRs, ingresses, deployments, pod "previous" logs, namespace events, etc.
+* ent is now able to sync the kubectl version of the current profile with the server version (`ent auto-align-kubectl`)
+* Added helper to fetch logs from bundle plugins (`ent prj be-log`).
+* In order to run node-based Entando tools, ent now installs and uses a private copy of node at user-level.
+* A jq internal dependency is now installed at user-level, privately to ent.
+* Added the command `ent pkg` (to manage packages installed at user-level) with initial support for jq and K9s.
+* Added the helpers `get bundle id` and `get plugin id` to `ent ecr` and `ent prj` in order to determine the names of the resources related to a bundle.
+* Added the command `ent gen-secret`  to `ent ecr` and `ent prj` in order to help generate Secrets for the plugins.
+* Added the `ent ecr cr` and `ent ecr deploy` subcommands.
+* ent now installs the entando-releases dependencies at release level, allowing multiple profiles to share them.
+* Fixed a bug that caused ent to git-ignore the "bundle" subdirs in the MFE branches.
+* `ent pbs-init` now accepts ssh/git URLs.
+* `ent check-env` now considers Docker and java mandatory dependencies (still skippable with "--lenient").
+* Several minor fixes are not listed here explicitly.
+
+### ENT Quickstart
+
+* Updated to support release 7.0.0.
+* Note that the default K3s version used by the entando-cli (ent) quickstart has been updated to v1.21.
+
+### Identity Provider (IdP)
+
+* Updated Keycloak to version 15.
+
+### Infrastructure
+
+* Includes a massive rework of the internal dependency management system wrt the concept of ProvidedCapability.
+* Added support for OKD4.8 and kube 1.21.
+* Added ComponentManager and Keycloak hardening.
+* Tuned Entando Platform timeouts and memory limits.
+* Moved from a HELM based deployment process to a pure CustomResource based deployment and installation process.
+* Installation parameters considered by the operator all moved to the entando-operator-config ConfigMap.
+* Better support for cloud DBMS.
+* Updated libraries (Jetty, Keycloak, Spring) and the entire compatibility matrix.
+* Dropped support for suffix-mode installation. Only single-hostname is supported now.
+* Temporarily dropped support for cluster-level operator installation.
```

Comment:
@w-caffiero-entando "temporarily"? What does this mean for the current OpenShift install guide which includes both cluster and project-level. If this won't be added back (I'm not at all sure of the priority) in a fixpack (7.0.x) then I would remove the word temporarily.

Reply:
@Lyd1aCla1r3 I discussed with @w-caffiero-entando , let's remove the word Temporarily. It may come back later but is not there in 7.0.0. Let's check for other usages of "temporary"